service/frontend/admin_handler.go

```diff
@@ -1117,40 +1116,8 @@ func (adh *AdminHandler) DescribeCluster(
 	}, nil
 }
 
-// ListClusters return information about temporal clusters
-// TODO: Remove this API after migrate tctl to use operator handler
-func (adh *AdminHandler) ListClusters(
-	ctx context.Context,
-	request *adminservice.ListClustersRequest,
-) (_ *adminservice.ListClustersResponse, retError error) {
-	defer log.CapturePanic(adh.logger, &retError)
-	if request == nil {
-		return nil, errRequestNotSet
-	}
-	if request.GetPageSize() <= 0 {
-		request.PageSize = listClustersPageSize
-	}
-
-	resp, err := adh.clusterMetadataManager.ListClusterMetadata(ctx, &persistence.ListClusterMetadataRequest{
-		PageSize:      int(request.GetPageSize()),
-		NextPageToken: request.GetNextPageToken(),
-	})
-	if err != nil {
-		return nil, err
-	}
-
-	var clusterMetadataList []*persistencespb.ClusterMetadata
-	for _, clusterResp := range resp.ClusterMetadata {
-		clusterMetadataList = append(clusterMetadataList, clusterResp.ClusterMetadata)
-	}
-	return &adminservice.ListClustersResponse{
-		Clusters:      clusterMetadataList,
-		NextPageToken: resp.NextPageToken,
-	}, nil
-}
 
 // ListClusterMembers
-// TODO: Remove this API after migrate tctl to use operator handler
```

Comment:
This comment was made somewhat in error. This api is still critical, not duplicated anywhere and still used by tdbg.